<launch>
  <node pkg="template" type="motion_detector.py" name="motion_detector" output="screen"/>
  <node pkg="rqt_image_view" type="rqt_image_view" name="rqt_image_view" args="/usb_cam/image_raw"/>
  <node pkg="rqt_plot" type="rqt_plot" name="rqt_plot" args="/usb_cam/motion_detection/diff"/>
</launch>